test/Rings/mpoly-graded-test.jl

```diff
@@ -107,7 +107,7 @@ end
         @test (RR in graded_rings) == Oscar.is_graded(RR)
         @test (RR in filtered_rings) == Oscar.is_filtered(RR)
         polys = _random_poly(RR, 4) # create 4 random polynomials
-        @test ngens(RR) == length(gens(RR))
+        @test ngens(RR) == ngens(RR)
```

Comment:
For this one test: No, you do not want to replace it here, as it tests ngens agains length(gens).

Reply:
Indeed, thanks!